src/ucp/api/ucp.h

```diff
@@ -1256,6 +1265,20 @@ typedef struct ucp_worker_attr {
 } ucp_worker_attr_t;
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief Tuning parameters for the comparison function @ref ucp_rkey_compare
```

Comment:
Please add full description in addition to `@brief`. We typically don't allow just brief.

Reply:
added description